src/pkg/bigbang/bigbang.go

```diff
@@ -0,0 +1,280 @@
+	}
+
+	bb := zarfHelmInstance.DownloadChartFromGit("bigbang")
+
+	zarfHelmInstance.ChartLoadOverride = bb
+
+	template, err := zarfHelmInstance.TemplateChart()
+	if err != nil {
+		return component, fmt.Errorf("unable to template BigBang Chart: %w", err)
+	}
+
+	subPackageURLS := findURLs(template)
+	repos[0] = fmt.Sprintf("%s@%s", repos[0], component.BigBang.Version)
+	repos = append(repos, subPackageURLS...)
+
+	component.Repos = repos
+
+	// Get all the images.  This might be omitted here once we have this logic more globally
+	// so that images are pulled from the chart annotations
+	err = importBigbangImageList(component.BigBang.Version)
+	if err != nil {
+		return component, fmt.Errorf("unable to import bigbang image list: %w", err)
+	}
+
+	images, err := GetImages(repos)
+	if err != nil {
+		return component, fmt.Errorf("unable to get bb images: %w", err)
+	}
+
+	// deduple
+	uniqueList := utils.Unique(images)
+
+	component.Images = append(component.Images, uniqueList...)
+
+	return component, nil
+}
+
+func findURLs(t string) []string {
+
+	// Break the template into separate resources
+	urls := make([]string, 0)
+	yamls, _ := utils.SplitYAML([]byte(t))
+
+	for _, y := range yamls {
+		// see if its a GitRepository
+		if y.GetKind() == "GitRepository" {
+			url := y.Object["spec"].(map[string]interface{})["url"].(string)
+			tag := y.Object["spec"].(map[string]interface{})["ref"].(map[string]interface{})["tag"].(string)
+			urls = append(urls, fmt.Sprintf("%v@%v", url, tag))
+		}
+	}
+
+	return urls
+}
+
+func GetImages(repos []string) ([]string, error) {
+
+	images := make([]string, 0)
+	for _, r := range repos {
+		// Don't try to get images for the bigbang repo
+		parts := strings.Split(r, "@")
+		if len(parts) != 2 {
+			continue
+		}
+
+		url := parts[0]
+		version := parts[1]
+
+		parts = strings.Split(url, "/")
+		packageRepo := parts[len(parts)-1]
+		packageRepo = strings.TrimSuffix(packageRepo, ".git")
+
+		images = append(images, BigBangImages[packageRepo][version]...)
+
+	}
+
+	return images, nil
+}
+
+func importBigBangFluxImageList(version string) error {
+	repo1, err := gitlab.NewClient("", gitlab.WithBaseURL("https://repo1.dso.mil/api/v4"))
+	if err != nil {
+		return fmt.Errorf("unable to create gitlab client: %w", err)
+	}
+
+	var rawFileOptions gitlab.GetRawFileOptions
+	rawFileOptions.Ref = &version
+
+	kustFile, _, err := repo1.RepositoryFiles.GetRawFile("2872", "base/flux/kustomization.yaml", &rawFileOptions, nil)
+	if err != nil {
+		return fmt.Errorf("unable to get flux kustomization file: %w", err)
+	}
+
+	var fluxKustomize kustypes.Kustomization
+
+	goyaml.Unmarshal(kustFile, &fluxKustomize)
+
+	if FluxImages[version] == nil {
+		FluxImages[version] = make([]string, 0)
+	}
+
+	for _, image := range fluxKustomize.Images {
+		FluxImages[version] = utils.Unique(append(FluxImages[version], fmt.Sprintf("%s:%s", image.NewName, image.NewTag)))
+	}
+	return nil
+}
+
+func importBigbangImageList(version string) error {
```

Comment:
Not sure if you want to rework this all now or wait until it feels more mature, but the epic for image annotations in charts was finished/closed today - https://repo1.dso.mil/groups/big-bang/-/epics/220#note_1067402

Reply:
I'd love to know if that truly is done. If so, we should just do that now and remove the custom logic.